analyses/cell-types-annotation/run-cell-types-annotation.R

```diff
@@ -0,0 +1,109 @@
+#################################################################################
+# This will run all scripts in the module
+#################################################################################
+# Load library
+suppressPackageStartupMessages({
+  library(yaml)
+  library(tidyverse)
+  library(celldex)
+  })
+
+#################################################################################
+# load config file
+configFile <- paste0("../../project_parameters.Config.yaml")
+if (!file.exists(configFile)){
+  cat("\n Error: configuration file not found:", configFile)
+  stop("Exit...")}
+
+# read `yaml` file defining the `params` of the project and strategy analysis
+yaml <- read_yaml(configFile)
+
+#################################################################################
+# Set up directories and paths to root_dir and analysis_dir
+root_dir <- yaml$root_dir
+analysis_dir <- file.path(root_dir, "analyses", "cell-types-annotation") 
+report_dir <- file.path(analysis_dir, "plots", "01_cell_types_annotation_SingleR") 
+
+################################################################################################################
+# Celldex references: These references are intended to be comparable to data from unique molecular identifier (UMI) protocols
+# where the expression values are less sensitive to differences in gene length. 
+# https://bioconductor.org/packages/3.18/data/experiment/vignettes/celldex/inst/doc/userguide.html
+# Download the database to use for the cell type annotation according to the species/project
+# Options: 
+# BlueprintEncodeData: bulk RNA-seq data for pure stroma and immune cells generated by Blueprint
+# DatabaseImmuneCellExpressionData: bulk RNA-seq samples of sorted cell populations (DICE reference)
+# HumanPrimaryCellAtlasData: publicly available microarray datasets derived from human primary cells. Most of the labels refer to blood subpopulations but cell types from other tissues are also available.
+# ImmGenData: microarray profiles of pure mouse immune cells
+# MonacoImmuneData: bulk RNA-seq samples of sorted immune cell populations
+# MouseRNAseqData: a collection of mouse bulk RNA-seq data sets downloaded from the gene expression omnibus
+# NovershternHematopoieticData: microarray datasets for sorted hematopoietic cell populations
+celldex_reference = yaml$celldex_reference_annotation_module
```

Comment:
In them `yaml` file it seems that the reference parameter is called `celldex_reference`, but here it is specified as `yaml$celldex_reference_annotation_module`.  Is this correct or should it be updated?

Reply:
Fixed here 852feeca38c6e88d7e32b271217fae5eed2cfd5b.